### PR TITLE
[CI:DOCS] fix updated links

### PR DESCRIPTION
**Pull request by fulldecent**
#### Does this PR introduce a user-facing change?

None

```release-note
Updates links to install documentation
```

### CODE REVIEW

**Comment by mheon** [comment]
Appears to have made the RPM build job angry. @baude I think you're the one who maintains that?

File: install.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# libpod Installation Instructions
```
Comment (vrothberg) [inline install.md:1]: I still see value in keeping this file and let it point to podman.io as it directs users arriving on GitHub to the right location.

**Review by vrothberg — COMMENTED** [review]
Other than the nit, LGTM

Thanks!

File: install.md (hunk 1 above)
Comment (rhatdan) [reply install.md:1]: Yes leave this file and put the link into it.

Comment (TomSweeneyRedHat) [reply install.md:1]: Yes please do leave the file, and that will likely cure your RPM test issue below. 
Otherwise LGTM

**Comment by fulldecent** [comment]
Restored file, rebased, squashed, signed-off

**Comment by rhatdan** [comment]
/approve
/lgtm
Thanks @fulldecent

**Pull request merged**